cmd/boost/deal_cmd.go

```diff
@@ -78,14 +78,19 @@ var dealFlags = []cli.Flag{
 		Value: true,
 	},
 	&cli.BoolFlag{
-		Name:  "fast-retrieval",
-		Usage: "indicates that data should be available for fast retrieval",
+		Name:  "keep-unsealed-copy",
+		Usage: "indicates that an unsealed copy of the sector should be available for fast retrieval",
 		Value: true,
 	},
 	&cli.StringFlag{
 		Name:  "wallet",
 		Usage: "wallet address to be used to initiate the deal",
 	},
+	&cli.BoolFlag{
+		Name:  "announce-to-ipni",
+		Usage: "indicates that deal index should be announced to the IPNI(Network Indexer)",
+		Value: true,
+	},
```

Comment:
These are new, never before existing CLI options: give them the same treatment as the protocol - explicitly asking for the less desirable thing
`--remove-unsealed-copy`
`--skip-ipni-announce`

Reply:
On the CLI we can set a default value more easily, so I would favour using a clearer name. To me boolean names are clearer when they're positive like "--announce-to-ipni" rather than negative "--skip-ipni-announce".

Reply:
Thanks Dirk and Riba! While the negative names (remove-unsealed-copy and skip-ipni-announce) are definitely annoying, I think we do want to keep the defaults of these booleans to false to align with general developer experience. In the case where an SP skips setting these flags all together, having the flags as false would be aligned with what they would expect. 

Unfortunately... if we want the defaults to be false, then the names will be negative... :(

Reply:
When a user doesn't supply these flags on the command line, they will default to the value we chosen for them.
In this case we have coded the parameter name to be `--announce-to-ipni` and the default value to be `true`.

Some example commands and the result:
Flag value | Example Command | Whether index will be announced
---------- | ------- | -----
`true` | `boost deal --announce-to-ipni=true` | `Yes`
`false` | `boost deal --announce-to-ipni=false` | `No`
Not set | `boost deal` | `Yes`

Note that the Command Line Interface is distinct from the [Propose Storage Deal Protocol](https://boost.filecoin.io/boost-architecture/libp2p-protocols#propose-storage-deal-protocol). Most SPs probably won't need to know of the protocol, they will just interact with the CLI tool.

Reply:
SP's doing deals with CLI are probably only testing and preparing to use the actual protocol though

Reply:
We talked sync and decided we will use the "negative" naming:
`--remove-unsealed-copy` (default false)
`--skip-ipni-announce` (default false)